dataset/get-metadata.go

```diff
@@ -16,6 +21,8 @@ type ClientError interface {
 	Code() int
 }
 
+var EDITION_CONFIRMED_STATE = "edition-confirmed"
```

Comment:
Yeah I agree this should be a constant instead of variable.

If the constant needs to be exported, then better to structure it like so: `const EditionConfirmedState = "edition-confirmed"` otherwise if it is not used outside package, it should be like `const editionConfirmedState = "edition-confirmed"`

Reply:
:+1: